ci/docker/runtime_functions.sh

```diff
@@ -591,6 +591,65 @@ build_docs() {
     popd
 }
 
+
+# Functions that run the nightly Tests:
+
+#Runs Apache RAT Check on MXNet Source for License Headers
+nightly_test_rat_check() {
+    set -ex
+    #This Test fails without changing permissions
```

Comment:
With what error?

Reply:
I get it for broken_link_checker too ->
`./tests/nightly/broken_link_checker_test/broken_link_checker.sh: Permission denied`

Reply:
This means that the file has not been committed with the correct file permissions. Try chmod 0755

Reply:
0755 works